tests/desktop_libreoffice_writer.pm

```diff
@@ -12,7 +12,7 @@ sub run {
     type_very_safely("writer\n");
     assert_screen('libreoffice_writer_main_window', 10);
 
-    type_string('hello world');
+    type_very_safely('hello world');
```

Comment:
! does one really get typos if one types too fast?

Reply:
Yeah, because OpenQA does its typing by sending key-down and key-up events over VNC, so if the timings of those get messed up (due to VNC latency, input handling latency on the SUT, or an Act of God), then you get repeated or swapped characters. :'-(